src/lightly_train/_task_models/dinov3_ltdetr_object_detection/task_model.py

```diff
@@ -500,7 +501,7 @@ def predict(
         x = transforms_functional.resize(x, self.image_size)
         x = x.unsqueeze(0)
 
-        labels, boxes, scores = self(x, orig_target_size=(h, w))
+        labels, boxes, scores = self(x, orig_target_size=torch.tensor([[h, w]]))
```

Comment:
Why did you have to modify the `predict` method? The ONNX export should only call `forward` and not `predict`.